src/Components/Navbar/navbar.js

```diff
@@ -16,15 +25,18 @@ const Navbar = () => {
         <Link to='/products'><a>products</a></Link>
         <Link to='/wishlist'><a>Wishlist</a></Link>
         <Link to='/cart'><a>Cart</a></Link>
```

Comment:
Why `<a>` inside link ??

Reply:
By mistake will correct that. Thankyou!!